### PR TITLE
added new design

### PR DESCRIPTION
**Pull request by ghost**
- I added border top hover thingy and shadow on the bottom of blog card for mobile screens.
- Added `transition-colors` on the navbar when hovered so it will fade smoothly.
- Fixed the highlight code on a code block for small screens. It will not work anymore if you have a really long one line code 😅
- Fixed the filename on the code block. Now it looks a bit smaller.

### CODE REVIEW

File: src/components/BlogCard.astro [hunk 1]
```diff
@@ -12,15 +12,15 @@ const fullDate = new Date(pubDate).toLocaleDateString("en", {
 ---
 
 <li
-  class="grid grid-cols-1 gap-2 sm:grid-cols-4 leading-relaxed rounded-xl border-zinc-700 text-zinc-400 transition-all"
+  class="grid grid-cols-1 gap-2 sm:grid-cols-4 leading-relaxed rounded-xl border-zinc-800 hover:border-zinc-600 border-t sm:border-none text-zinc-400 transition-all shadow-xl sm:shadow-none"
```
Comment (kevinzunigacuellar) [inline src/components/BlogCard.astro]: I think I will just keep it simple for mobile
```suggestion
  class="grid grid-cols-1 gap-2 sm:grid-cols-4 leading-relaxed rounded-xl text-zinc-400"
```

File: src/components/BlogCard.astro [hunk 2]
```diff
@@ -12,15 +12,15 @@ const fullDate = new Date(pubDate).toLocaleDateString("en", {
...
 >
   <time
     datetime={pubDate}
     class="hidden sm:block text-sm tracking-wide text-zinc-400 mt-6"
     >{fullDate}
   </time>
   <a
-    class="col-span-3 flex flex-col gap-2 sm:gap-4 sm:p-5 sm:hover:bg-zinc-800 transition-colors rounded-lg"
+    class="col-span-3 flex flex-col gap-2 sm:gap-4 p-5 sm:hover:bg-zinc-800 transition-colors rounded-lg"
```
Comment (kevinzunigacuellar) [inline src/components/BlogCard.astro]:
```suggestion
    class="col-span-3 flex flex-col gap-2 sm:gap-4 sm:p-5 sm:hover:bg-zinc-800 transition-colors rounded-lg"
```

File: src/styles/syntaxHighlight.css [hunk 3]
```diff
@@ -3,7 +3,7 @@ div[data-rehype-pretty-code-fragment] {
 }
 
 div[data-rehype-pretty-code-title] {
-  @apply inline-block px-4 bg-zinc-800 text-base m-0 pt-0.5 pb-1 text-blue-400
+  @apply inline-block px-4 bg-zinc-800 text-sm m-0 pt-1.5 pb-1.5 text-blue-400
```
Comment (kevinzunigacuellar) [inline src/styles/syntaxHighlight.css]: looks amazing 💜
```suggestion
  @apply inline-block px-4 bg-zinc-800 text-sm m-0 py-1.5 text-blue-400
```

File: src/styles/syntaxHighlight.css [hunk 4]
```diff
@@ -19,5 +19,5 @@ div[data-rehype-pretty-code-fragment] pre code .line {
 }
 
 div[data-rehype-pretty-code-fragment] .highlighted {
-  @apply bg-green-600/20
+  @apply bg-green-600/20 w-[800px]
```
Comment (kevinzunigacuellar) [inline src/styles/syntaxHighlight.css]: I think I will just hide it until I figure this one out
```suggestion
  @apply sm:bg-green-600/20 
```

**Review by kevinzunigacuellar — COMMENTED** [review]
thank you 💜, I appreciate this a lot

**Comment by ghost** [comment]
Thanks! It's ok if you want to keep the blog card simple, it's just my suggestion 😅.

**Pull request merged**